src/atomate2/common/jobs/qha.py

```diff
@@ -0,0 +1,126 @@
+"""Jobs for running qha calculations."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from jobflow import Flow, Response, job
+
+from atomate2.common.schemas.phonons import PhononBSDOSDoc
+from atomate2.common.schemas.qha import PhononQHADoc
+
+if TYPE_CHECKING:
+    from pymatgen.core.structure import Structure
+
+    from atomate2.common.flows.phonons import BasePhononMaker
+
+logger = logging.getLogger(__name__)
+
+
+@job(
+    data=[PhononBSDOSDoc],
+)
+def get_phonon_jobs(phonon_maker: BasePhononMaker, eos_output: dict) -> Flow:
+    """
+    Start all relevant phonon jobs.
+
+    Parameters
+    ----------
+    phonon_maker: .BasePhononMaker
+        Maker to start harmonic phonon runs.
+    eos_output: dict
+        Output from EOSMaker
+
+    """
+    phonon_jobs = []
+    outputs = []
+    for istructure, structure in enumerate(eos_output["relax"]["structure"]):
+        phonon_job = phonon_maker.make(structure)
```

Comment:
If possible, it would be nice to pass the prev_dir from the relaxation? That way we can make use of `auto_ispin` and make the calculations cheaper.

Reply:
Did this now by extending the EOS dictionaries.